packages/oui-guide-menu/guide-menu.less

```diff
@@ -0,0 +1,43 @@
+@import './_normalize';
+@import './_variables';
+
+.oui-guide-menu {
+  #oui > .guide-menu-normalize();
+
+  display: block;
```

Comment:
Should be more an `inline-block` component, like the action-menu and the dropdown :)

Reply:
oki